histomicstk/cli/utils.py

```diff
@@ -49,7 +49,8 @@ def get_stain_matrix(args, count=3):
     return np.array([get_stain_vector(args, i + 1) for i in range(count)]).T
 
 
-def segment_wsi_foreground_at_low_res(ts, lres_size=2048):
+def segment_wsi_foreground_at_low_res(
+        ts=None, lres_size=2048, invert_image=False, frame=None, default_img_inversion=False):
```

Comment:
We _have_ to pass `ts`, so the default shouldn't be `None`.

Reply:
Great suggestion, I missed that one. I've changed the default value for ts.